protocol/x/prices/keeper/market_price.go

```diff
@@ -174,10 +174,20 @@ func (k Keeper) GetMarketIdToValidIndexPrice(
 	ret := make(map[uint32]types.MarketPrice)
 	for _, marketParam := range allMarketParams {
 		if indexPrice, exists := marketIdToValidIndexPrice[marketParam.Id]; exists {
+			exponent, err := k.GetExponent(ctx, marketParam.Pair)
```

Comment:
While we don't expect this call to fail, if it does no prices will be returned. Since we already do an error log, perhaps we should just skip this market and continue for the next market rather than returning `nil`?

Reply:
good point, i should've thought about this more carefully